### PR TITLE
Added WebRTC demo

### PR DESCRIPTION
**Pull request by blagojce95**
This PR contains a demo for WebRTC as a backend for p2p communication. It is only tested locally, we plan to test it using a google cloud instance as a signaling server. This solution should also be able to connect peers which are behind NATs.

### CODE REVIEW

File: WebRTC_backend/textchat/client.js [hunk 1]
```diff
@@ -0,0 +1,210 @@
+//our username 
+var name; 
+var connectedUser; 
+
+//connecting to our signaling server 
+var conn = new WebSocket('ws://52.200.7.179:55535');
```
Comment (mmilenkoski) [inline WebRTC_backend/textchat/client.js:6]: We should change this line to `var conn = new WebSocket('ws://localhost:9090');` for the local version

**Comment by martinjaggi** [comment]
thanks! as this one is 'pure' WebRTC, how does it relate to the peerjs ones (which also use WebRTC as backend) #16 and #17

**Comment by martinjaggi** [comment]
we can close this and/or add it to the /experimental/  folder

**Comment by mmilenkoski** [comment]
Closing this because it was moved to `/experiments` with 73b133cc0314c536c83d947674ec89b716baa1b7.

**Pull request closed without merging**